java/flight/flight-core/src/test/java/org/apache/arrow/flight/TestClientMiddleware.java

```diff
@@ -84,6 +84,8 @@ public void testMultiValuedHeaders() {
         });
     // The server echoes the headers we send back to us, so ensure all the ones we sent are present with the correct
     // values in the correct order.
+    EXPECTED_BINARY_HEADERS.put("x-binary-bin", Arrays.asList(new byte[] {0}, new byte[]{1}));
+    EXPECTED_TEXT_HEADERS.put("x-text", Arrays.asList("foo", "bar"));
```

Comment:
Hmm, inline the declarations here too if they're only used in this test? Else put the initialization in a `static` block

Reply:
Changed